nix/default.nix

```diff
@@ -30,18 +32,10 @@ let
           ];
         });
         systemd = (static pkg.systemd).overrideAttrs(x: {
+          outputs = [ "bin" "out" "lib" ];
           mesonFlags = x.mesonFlags ++ [
             "-Dstatic-libsystemd=true"
           ];
```

Comment:
@saschagrunert seems like this changes for systemd 246 working with crun (https://github.com/containers/crun/pull/485/files#diff-990d93ae620a3c4824fd6544548bf806L29-L37) but not here.

I think most likely related to our static pcre overlays. May I have some hints for this?

Reply:
This looks good, doesn’t it? 👀

Reply:
I think we needed the PCRE overlay because of building glib statically, right?

Hmmm, this build error looks indeed strange:
```
PermissionError: [Errno 13] Permission denied: '//02qcpld1y6xhs5gz9bchpxaw0xdhmsp5dv88lh25r2ss44kh8dxz'
```

Reply:
Fixed, only `outputs = [ "out" "lib" ];` could specify :-S